src/test/java/core/basesyntax/services/ReportServiceImplTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ReportServiceImplTest {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
```

Comment:
It may be a redundant constant. Andrew told me about it in previous task.

Reply:
i use LINE_SEPARATOR for resolve DRY problem, the constant helps not to repeat

Reply:
DRY is not the case here - you don't have to use any constants to avoid DRY violation in your tests

read carefully what is DRY principle - multiple calls to `System.lineSeparator()` don't violate it